src/views/examples/all-components/index.njk

```diff
@@ -2,69 +2,37 @@
 
 {% block content %}
 
-  {% include "../components/breadcrumb/breadcrumb.njk" %}
+  {% include "../../../components/breadcrumbs/breadcrumbs.njk" %}
```

Comment:
do we need full relative paths here? it can be `{% include "breadcrumbs/breadcrumbs.njk" %}` since we [include the components path in Nunjucks views](https://github.com/alphagov/govuk-frontend/blob/master/app.js#L14)